.env.benefit-backend.example

```diff
@@ -11,11 +11,10 @@ CORS_ALLOW_ALL_ORIGINS=1
 # if changing DUMMY_COMPANY_FORM_CODE, also set NEXT_PUBLIC_MOCK_FLAG=1
 # DUMMY_COMPANY_FORM_CODE=16
 # NEXT_PUBLIC_MOCK_FLAG=1
-# DISABLE_AUTHENTICATION=1
 # DISABLE_TOS_APPROVAL_CHECK=1
 
 ## Authentication
-DISABLE_AUTHENTICATION=0
+NEXT_PUBLIC_MOCK_FLAG=0
```

Comment:
Does not describe anything what it does. Authentication flag is replaced with something I don't understand. It does have a readme, which is good, but the variable could also be renamed

Reply:
Ok, didn't rename it. I did however make a ticket [HL-575](https://helsinkisolutionoffice.atlassian.net/browse/HL-575) about the possibility of renaming the variable to be more intuitive/descriptive: